sky/cli.py

```diff
@@ -1706,13 +1722,51 @@ def status(all: bool, refresh: bool, show_spot_jobs: bool, clusters: List[str]):
                           show_all=False,
                           limit_num_jobs_to_show=not all,
                           is_called_by_user=False))
-        click.echo(f'{colorama.Fore.CYAN}{colorama.Style.BRIGHT}Clusters'
-                   f'{colorama.Style.RESET_ALL}')
+        if ip:
+            if len(clusters) != 1:
+                with ux_utils.print_exception_no_traceback():
+                    plural = 's' if len(clusters) > 1 else ''
+                    raise RuntimeError(
+                        _STATUS_IP_CLUSTER_NUM_ERROR_MESSAGE.format(
+                            cluster_num=len(clusters),
+                            plural=plural,
+                            verb='specified'))
```

Comment:
UX nit: When no clusters are specified, we get the following message:

```
(base) ➜  sky-experiments git:(sky-status-ip) ✗ sky status --ip
RuntimeError: 0 cluster specified. Please specify exactly one existing cluster to show its IP address.
Usage: `sky status --ip <cluster>`
```

Instead of `0 cluster specified.` can we say `No cluster specified.`?